src/main/java/appeng/items/tools/powered/ToolWirelessTerminal.java

```diff
@@ -56,6 +70,50 @@ public ToolWirelessTerminal()
 	{
 		super( AEConfig.instance.wirelessTerminalBattery, Optional.<String>absent() );
 		this.setFeature( EnumSet.of( AEFeature.WirelessAccessTerminal, AEFeature.PoweredTools ) );
+
+		if( Platform.isClient() )
+		{
+			MinecraftForgeClient.registerItemRenderer( this, new ToolWirelessTerminalRender() );
+		}
+	}
+
+	public static AEColor getColor( ItemStack item )
+	{
+		final NBTTagCompound c = item.getTagCompound();
+		if( c != null && c.hasKey( "color" ) )
+		{
+			String color = c.getString( "color" );
+			if( color != null )
+			{
+				try {
+					return AEColor.valueOf( color );
+				}
+				catch( final Throwable e )
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Catch Exception instead of Throwable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1181)